HeterogeneousCore/AlpakaMath/interface/deltaPhi.h

```diff
@@ -0,0 +1,35 @@
+#ifndef HeterogeneousCore_AlpakaMath_deltaPhi_h
+#define HeterogeneousCore_AlpakaMath_deltaPhi_h
+
+#include <alpaka/alpaka.hpp>
+
+namespace cms::alpakatools {
+
+  // reduce to [-pi,pi]
+  template <typename TAcc, std::floating_point T>
+  ALPAKA_FN_HOST_ACC inline T reduceRange(TAcc const& acc, T x) {
+    constexpr T o2pi = T{1.} / (T{2.} * std::numbers::pi_v<T>);
+    if (alpaka::math::abs(acc, x) <= std::numbers::pi_v<T>)
+      return x;
+    T n = alpaka::math::round(acc, x * o2pi);
```

Comment:
Just curious: why not simply
```suggestion
    T n = alpaka::math::round(acc, x / (T{2} * std::numbers::pi_v<T>));
```
or
```suggestion
    T n = alpaka::math::round(acc, x * std::numbers::inv_pi_v<T> * T{0.5});
```
?

Reply:
As initially mentioned, this was meant to be an alpaka-adjusted copy of the `reco::reduceRange` function
https://github.com/cms-sw/cmssw/blob/a475844e9e68627f9a367b74a2beb482f965d3a7/DataFormats/Math/interface/deltaPhi.h#L18-L24
hence I didn't apply any other simplifications, apart from the ones suggested in this PR.

Reply:
ok